bin/NativeTests/JsRTApiTest.cpp

```diff
@@ -1478,17 +1478,18 @@ namespace JsRTApiTest
     {
         JsValueRef result = JS_INVALID_REFERENCE, callbackState = JS_INVALID_REFERENCE, exception = JS_INVALID_REFERENCE;
         JsValueType cbStateType = JsUndefined;
+        const wchar_t *script = _u("new Promise((res, rej) => res()).then(() => 1)");
```

Comment:
Honestly this method is a little bit of a mess.  It's Windows only and the header uses wchar_t and the implementation uses "WCHAR".  I'd say just leave it for now.